### PR TITLE
Responsiveness improved, social icons moved below the side-bar and side-bar turned to a flex object.  

### PR DESCRIPTION
**Pull request by shashwat1002**
Issue numbers mentioned in the commit messages do not correspond to the issue numbers of fossasia/2015.fossasia.org they are as follows:
preview link: https://shashwat1002.github.io/2014.fossasia.org/
"#3" in the messages refers to #69 
"#2" refers to #71 
"#1" refers to #70

### CODE REVIEW

**Comment by nzec** [comment]
@shashwat1002 Give a preview link using GitHub Pages. And also squash commits as well as resolve merge conflicts.

Rest all looks good!

File: _config.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ description: > # this means to ignore newlines until "baseurl:"
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
 baseurl: "" # the subpath of your site, e.g. /blog/
-url: "http://akshattripathi.github.io/fossasia14/" # the base hostname & protocol for your site
+url: "https://shashwat1002.github.io/2014.fossasia.org/" # the base hostname & protocol for your site
```
Comment (nzec) [inline _config.yml]: Add the actual path of the website.

Which is `http://fossasia.github.io/2014.fossasia.org/`

**Comment by shashwat1002** [comment]
Does this work?

**Comment by shashwat1002** [comment]
@duttaditya18  @invinciblycool , could you please review my pull request?

**Review by harshit98 — APPROVED** [review]

**Comment by harshit98** [comment]
Excellent work @shashwat1002. You've improved a lot of UI design. Keep it up!

**Pull request merged**